phpBB/phpbb/storage/adapter/local.php

```diff
@@ -331,4 +337,12 @@ public function file_image_height($path)
 	{
 		return $this->image_dimensions($path);
 	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function get_link($path)
+	{
+		return generate_board_url() . $this->root_path . $path;
```

Comment:
generate_board_url() does not need a root_path

Reply:
Yes, sorry, it was just path, i forgot to change it